crates/matrix-sdk-base/src/read_receipts.rs

```diff
@@ -83,7 +106,7 @@ impl RoomReadReceipts {
     ///
     /// Returns whether a new event triggered a new unread/notification/mention.
     #[inline(always)]
-    fn update_for_event(&mut self, event: &SyncTimelineEvent, user_id: &UserId) -> bool {
+    fn account_event(&mut self, event: &SyncTimelineEvent, user_id: &UserId) -> bool {
```

Comment:
When you say "account_event" do you mean "account for event" (meaning something like "update my records based on the info in this event")? If so I would find "process_event" more idiomatic I think.